CONTRIBUTING.md

```diff
@@ -1,10 +1,10 @@
 <img src="Assets/ProjectBanner.png" alt="XPlat Windows APIs" />
 
-# Contributing to XPlat Windows APIs
+# Contributing
 
-These guidelines are designed to help you as a contributor to bring your code and skills to the project to make it better.  
+These guidelines are designed to help you as a contributor to bring your code and skills to the XPlat Windows API project to make it better.  
```

Comment:
Should this read XPlat Windows APIs project?

Reply:
Updated to read as XPlat Windows APIs.